docs/design/coreclr/botr/readytorun-format.md

```diff
@@ -428,6 +485,30 @@ The module override index translation algorithm is as follows (**ILAR** = *the n
 
 **TODO**: document attribute presence encoding
 
+**Note:** This is a per-assembly section. In single-file R2R files, it is pointed to directly by the
+main R2R header; in composite R2R files, each component module has its own attribute presence
+section pointed to by the `READYTORUN_SECTION_ASSEMBLIES_ENTRY` core header structure.
+
+## READYTORUN_SECTION_ASSEMBLIES (v3.2+)
+
+This image-wide section is only present in the main R2R header of composite R2R files. It is an
+array of the entries `READYTORUN_SECTION_ASSEMBLIES_ENTRY` parallel to the indices in the manifest metadata
+AssemblyRef table in the sense that it's a linear table where the row indices correspond to the
+equivalent AssemblyRef indices. Just like in the AssemblyRef ECMA 335 table, the indexing is
+1-based (the first entry in the table corresponds to index 1).
+
+```C++
+struct READYTORUN_SECTION_ASSEMBLIES_ENTRY
+{
+    IMAGE_DATA_DIRECTORY CorHeader;         // Input MSIL metadata COR header
+    IMAGE_DATA_DIRECTORY AvailableTypes;    // Available types table
+    IMAGE_DATA_DIRECTORY MethodEntrypoints; // Method entrypoint table
+};
```

Comment:
And speaking of

> Each READYTORUN_SECTION_ASSEMBLIES_ENTRY record points to a READYTORUN_CORE_HEADER variable-length structure representing sections specific to the particular assembly.

Where exactly is `READYTORUN_CORE_HEADER`?

Reply:
Around line 95 in the readytorun-format.md file.

Reply:
No, I was asking where the record "points to" that header. I see that you have updated this structure in the last commit so it looks good now. Thanks!